### ISSUE TITLE
android: game looks too dark

### ISSUE THREAD

**Issue by ozkriff** [ozkriff/zoc#200]
После перехода на gfx (и если вбить костыли в cgmath, что бы обойти `repr(C, packed)`-ошибки) картинка на андроиде стала очень темной:

![](http://i.imgur.com/yZ8t5Sr.png)

vs

![](http://i.imgur.com/BWnU9pK.png)

**Comment by ozkriff** [comment]
Кварк предложил попробовать сунуть glutin `with_srgb(Some(true))`.

**Comment by ozkriff** [comment]
Нет, простое использование `with_srgb(Some(true))` не помогает, надо копать глубже (видимо, лезть в кишки glutin).

---

 `with_srgb(Some(true))` haven't help :( Looks like I`ll have to dig into glutin android port.

**Comment by snuk182** [comment]
Not experiencing this, all colors present. 
Honor 6 Plus, Android 6.0.
